### ISSUE TITLE
Issue with install, usage, and fakexrandr-manage show-available

### ISSUE THREAD

**Issue by hackall360** [phillipberndt/fakexrandr#54]
Heres my exact steps:
Re-installed xrandr, and x11, and all related librarys.
I am on Raspberry Pi 3 model B, running raspbian.
When i ran sudo make i got this, i believe its successful install:
pi@raspberrypi:~/Documents/fakexrandr $ sudo make
cc -O2 -fPIC -shared -o libXrandr.so libXrandr.c -ldl
[ -e libXrandr.so.2 ] || ln -s libXrandr.so libXrandr.so.2
[ -e libXinerama.so.1 ] || ln -s libXrandr.so libXinerama.so.1

And when i ran sudo make install:
pi@raspberrypi:~/Documents/fakexrandr $ sudo make install
TARGET_DIR=`sed -nre 's/#define FAKEXRANDR_INSTALL_DIR "([^"]+)"/\1/p' config.h`; \
[ -d $TARGET_DIR ] || exit 1; \
install libXrandr.so $TARGET_DIR; \
ln -s libXrandr.so $TARGET_DIR/libXrandr.so.2 || true; \
ln -s libXrandr.so $TARGET_DIR/libXinerama.so.1 || true; \
ldconfig
ldconfig: /usr/lib/arm-linux-gnueabihf/libXinerama.so.1 is not a symbolic link

ldconfig: /usr/lib/arm-linux-gnueabihf/libXrandr.so.2 is not a symbolic link

ldconfig: /usr/lib/libXinerama.so.1 is not a symbolic link

ldconfig: /usr/lib/libXrandr.so.2 is not a symbolic link

install fakexrandr-manage.py /usr/bin/fakexrandr-manage

When running fakexrandr-manage help:
pi@raspberrypi:~ $ fakexrandr-manage
/usr/bin/fakexrandr-manage:37: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk
Syntax: fakexrandr-manage <gui|dump-config|show-available|clear-config|
                           set-config>

Available commands:
  gui
    Run the GTK based gui
  dump-config
   Dump the configuration file in a parseable format to the console. Different
   configurations are separated by an empty line.
  show-available
   Query XRandR and show outputs for which a configuration could be created.
  clear-config
   Remove all stored configurations
  set-config
   Load configurations from the standard input and merge them into the
   configuration file

Configuration format:
  The CLI configuration format follows sh syntax and defines the variables NAME,
  EDID, WIDTH, HEIGHT and SPLITS. SPLITS is a string describing how an output
  shall be split. It starts by one of the letters H, V or N, describing the
  kind of split. H means horizontal, V vertical and N no split. Separated by a
  space follows the pixel position of the split. Again separated by a space
  follow the two sub-configurations of the left/right or top/bottom halves. Any
  additional white-space besides a single space is optional any only serves
  better readibility. dump-config indents sub-configurations to this end.
  If SPLITS equals N, a configuration is discarded upon saving it.
 

And finally when running show available:
pi@raspberrypi:/ $ sudo fakexrandr-manage show-available
/usr/bin/fakexrandr-manage:37: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk

Any clue what would be wrong, if i was running just Debian, as Debian and raspbian are very close bu different, what would i need to install or do to fix this issue? 
I am attempting some raspberry pi DIY projects requiring dual screen setups on one screen

I relise there are 2 related topics, but they dont have as much detail as i feel was provided here.

If anyone can help, please do.

**Comment by akirak** [comment]
I encountered the exact same error on WSL (Ubuntu on Windows 10 with vcxsrv as a server):

```
/usr/bin/fakexrandr-manage:37: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk
```

**Comment by hackall360** [comment]
Then i must wait for the author to add compatibility for arm devices, or create compatibility myself, but i dont have time for that

**Comment by akirak** [comment]
I don't think it has anything to do with ARM, since I encountered the same error on Windows for Subsystem for Linux (WSL) on an Intel device.

**Comment by phillipberndt** [comment]
Neither arm nor wsl are supported. I'll be happy to merge any PRs, but I don't actively develop fakexrandr myself anymore (since I don't have a setup where I need it anymore)

**Comment by hackall360** [comment]
Alright then, it is not needed for my setup anymore either, as I have
abandoned my previous project that I planned on using it in. Thank you for
your response.
